nextflow.config

```
@@ -236,13 +238,13 @@ profiles {
     }
     docker {
         docker.enabled         = true
-        docker.userEmulation   = true
         conda.enabled          = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
         apptainer.enabled      = false
+        runOptions             = '-u $(id -u):$(id -g)'
```

```suggestion
        docker.runOptions             = '-u $(id -u):$(id -g)'
```

There was an error in the initial template and this appears to be the source of the geNomad database issues (the issue was docker specific)